pkg/ebpf/ksyms.go

```diff
@@ -3,18 +3,20 @@ package ebpf
 import (
 	"bufio"
 	"os"
-	"strings"
+	"sort"
 
+	"github.com/DataDog/datadog-agent/pkg/process/util"
 	"github.com/pkg/errors"
 )
 
 // VerifyKernelFuncs ensures all kernel functions exist in ksyms located at provided path.
 func VerifyKernelFuncs(path string, requiredKernelFuncs []string) ([]string, error) {
 	// Will hold the found functions
```

Comment:
Nit: it looks like this comment needs to be updated

Reply:
Good catch 🙇‍♂️